### DISCUSSION TITLE
The Plan Of Task Level Lineage

### DISCUSSION THREAD

**Discussion by shangeyao** [apache/dolphinscheduler#7744] [Ideas]
Hello folks,there are about 2000 + workflows in our production environment. Once the workflow in the intermediate link is abnormal, it is difficult for the business party to quickly locate the impact on the downstream. Through communication with zhongjiajie @zhongjiajie , I think we need to add task level lineage, because our current workflow relationship is not easy to use.
Do you have any comments or suggestions?

**Comment by zhongjiajie** [comment]
For community,
This discussion is related to https://github.com/apache/dolphinscheduler/issues/7156 which @boy-xiaozhang WIP and will add table like below

![image](https://user-images.githubusercontent.com/15820530/147800436-705c21b7-fa92-4121-9e2c-354905ae0bb8.png)


For now and issue only support SQL task type, and maybe it could be expanded to all task type later. Maybe we should discuss in this thread
